randovania/games/dread/logic_database/Ghavoran.txt

```diff
@@ -2061,46 +2061,69 @@ Extra - asset_id: collision_camera_026
   * Extra - tile_types: ('WEIGHT',)
   > Event - Golzuna
       All of the following:
-          Any of the following:
-              # Requirements for dodging attacks. Not needed if at least beginner combat.
-              Combat (Beginner)
+          All of the following:
+              # Requirements for dodging attacks.
               All of the following:
                   # Dodge its slam attack
                   Can Slide
                   Any of the following:
                       # Dodge its zappy attacks
-                      Use Spin Boost
-                      Flash Shift and Wall Jump (Beginner)
+                      Space Jump
+                      All of the following:
+                          Combat (Beginner)
+                          Flash Shift or Use Spin Boost
```

Comment:
My first thought here is that there might be a world where having both Flash Shift and Spin Boost could live at a lower Combat difficulty than each of them individually, but I don't think it's very important.

However, I'm not sure if simply `Flash Shift` is enough here, maybe you need to also require a certain number of Flash Shift Upgrades.

Reply:
Flash Shift alone goes the whole arena. Although i suppose just having Flash Shift could mean you might only have a single flash, if you set the main to not give any upgrades.